### DISCUSSION TITLE
Channel 8 kurdish

### DISCUSSION THREAD

**Discussion by mamosa14** [iptv-org/community#564] [Q&A]
Hi , can someone help get this stream to work for me https://channel8.com/english/live-tv
 When i get the stream out i get this https://live.channel8.com/Channel8-Kurdish/index.fmp4.m3u8 but its not workin on vlc …

**Comment by diaqad** [comment]
https://live.channel8.com/Channel8-Kurdish/tracks-a1/index.fmp4.m3u8

**Reply by mamosa14** [reply]
It seems that it works now in vlc as the Security check is disabled.
